stable/pipelines/templates/buildplane-config-aws.yaml

```diff
@@ -7,9 +7,9 @@ metadata:
     {{- include "pipelines.labels" . | nindent 4 }}
 data:
   provider: aws
-  accountId: {{ .Values.buildPlane.dynamic.customer.accountId | quote }}
-  nodePoolName: {{ .Values.buildPlane.dynamic.customer.nodePoolName | quote }}
-  nodelimit: {{ .Values.buildPlane.dynamic.customer.nodelimit | quote }}
+  accountId: {{ default "dummy" .Values.buildPlane.dynamic.customer.accountId | quote }}
```

Comment:
Should this really be here? It doesn't look too good...

Reply:
This was suggested by @rimusz.
Should we change this @rimusz ?